gui/src/api.js

```diff
@@ -35,6 +35,8 @@ const post = (path, body, shouldStringify = true) => {
     .then(response => {
       if (response.status === 401) {
         throw new AuthenticationError(response)
+      } else if (response.status >= 500 && response.status < 600) {
```

Comment:
You can have a status code >= 600?

optional nitpick: Remove the `< 600` clause. We sure want to raise an error if we get error 666.

Reply:
Lol. Are there status codes above 599?